distributionviewer/core/templates/distributionviewer/login.html

```diff
@@ -15,6 +15,14 @@
         window.location.pathname = '/permission-denied/';
       </script>
     {% endif %}
+    <script>
```

Comment:
Nope. Since the bundle is loaded on *index.html* and *react-ga* is being used inside the bundle, it would be redundant. At first I thought we could paste this into *index.html* instead of using react-ga, but that would show all hits as going to / since JavaScript manipulates history.